tasks/alerting.py

```diff
@@ -27,47 +27,51 @@ def send_alert_discord(context):
 
     # Send Alert
     env = Variable.get("ENV")
-    payload = {
-        "content": "",
-        "embeds": [
-            {
-              "title": f"{env} - Airflow Alert - Task failure",
-              "color": 1127128,
-              "url": f"https://airflow.test.bi.status.im/dags/{dag_name}",
-              "fields": [
-                {
-                  "name": "DAG",
-                  "value": dag_name,
-                  "inline": True
-                },
-                {
-                  "name": "ENV",
-                  "value": env,
-                  "inline": True
-                },
-                {
-                  "name": "TASK",
-                  "value": task_name,
-                  "inline": True
-                },
-                {
-                  "name": "ERROR",
-                  "value": error_message,
-                  "inline": True
-                },
-                {
-                  "name": "Date",
-                  "value": execution_date,
-                  "inline": True
-                }
-          ]
-        }
-      ]
-    };
-    webhook = HttpHook(http_conn_id="discord")
-    response = webhook.run(
-            endpoint=Variable.get("DISCORD_WEBHOOK"),
-            headers={"Content-type":"application/json"},
-            data=json.dumps(payload)
-        )
-    return response
+    # Only send the alert if in "prod" env
+    if env == "prod":
```

Comment:
It's better to do the opposite : 

```suggestion
    if env == "test":
            return "Not in 'prod' environment, alert not sent."
```
That's cleaner than to have big if block.